app/controllers/media/objects_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Media::ObjectsController < ApplicationController
+  before_action :set_media_object, only: [:show, :edit, :update, :destroy]
+  skip_authorization_check
+
+  # GET /media/objects/1
+  # GET /media/objects/1.json
+  def show
+    style = params[:style]
+    send_file @media_object.item.path(style), type: @media_object.item_content_type, disposition: 'inline'
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
```

Comment:
I think we can get rid of all the comments in this file. Did this come along with a gem or something? I don't think this was part of the issue spec?